src/components/helper-methods.js

```diff
@@ -12,8 +12,9 @@ export default {
     }
   },
 
-  getBaseProps(props, defaultStyles) {
-    const calculatedValues = this.getCalculatedValues(props, defaultStyles);
+  getBaseProps(props, defaultStyles, defaultColorScale) {
```

Comment:
Can you refactor this so that instead of `defaultStyles` and `defaultColorScale` you have one big object `fallbackProps` with `fallbackProps.style` etc. I can imagine adding to this, and it would be nice to keep the function signature the same here and in `victory-chart`